src/BootstrapTooltipContext.js

```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
         if (this._contextObj) {
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                callback: () => this.setTooltipTextAttribute()
+                callback: lang.hitch(this, (guid) => {
+                    this.setTooltipTextAttribute()
+                })
             });
 
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                attr: this.tooltipMessageAttribute,
-                callback: () => this.setTooltipTextAttribute()
+                attr: this._attribute,
+                callback: lang.hitch(this, (guid, attr, attrValue) => {
```

Comment:
What's the purpose here with `hitch` and unused parameters? Seems like this isn't necessary.

Reply:
https://dojotoolkit.org/reference-guide/1.7/dojo/hitch.html

Reply:
Arguments removed